source/adios2/engine/dataman/DataManWriter.cpp

```diff
@@ -102,39 +103,32 @@ void DataManWriter::Init()
                                  }) == s.end();
         };
 
-//        json jmsg;
-//        for (const auto &i : m_IO.m_Parameters)
-//        {
-//            if (lf_IsNumber(i.second))
-//            {
-//                jmsg[i.first] = std::stoi(i.second);
-//            }
-//            else
-//            {
-//                jmsg[i.first] = i.second;
-//            }
-//        }
-//        jmsg["stream_mode"] = "sender";
-//        m_Man.add_stream(jmsg);
+        //        json jmsg;
+        //        for (const auto &i : m_IO.m_Parameters)
+        //        {
+        //            if (lf_IsNumber(i.second))
+        //            {
+        //                jmsg[i.first] = std::stoi(i.second);
+        //            }
+        //            else
+        //            {
+        //                jmsg[i.first] = i.second;
+        //            }
+        //        }
+        //        jmsg["stream_mode"] = "sender";
+        //        m_Man.add_stream(jmsg);
 
         int n_Transports = 1;
         std::vector<Params> para(n_Transports);
```

Comment:
Just do `std::vector<Params>parameters (1);` I usually use plural names (e.g. parameters) for containers, so in a for range loop we can do something like this:
```
for( const auto& parameter : parameters )
{
}
```